omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-debian10-ice3.6.rst

```diff
@@ -0,0 +1,259 @@
+.. walkthroughs are generated using ansible, see 
+.. https://github.com/ome/omeroweb-install
+
+OMERO.web installation on Debian 10 and IcePy 3.6
+=================================================
+
+Please first read :doc:`../../server-debian10-ice36`.
+
+
+This is an example walkthrough for installing OMERO.web in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **omero-web system user** and define the main OMERO.web configuration options as environment variables. Since 5.6, a new :envvar:`OMERODIR` variable is used, you should first unset :envvar:`OMERO_HOME` (if set) before beginning the installation process.
+
+
+**The following steps are run as root.**
+
+If required, first create a local system user omero-web and create directory::
+
+    useradd -m omero-web
+
+    mkdir -p /opt/omero/web/omero-web/etc/grid
+    chown -R omero-web /opt/omero/web/omero-web
+
+
+
+Installing prerequisites
+------------------------
+
+**The following steps are run as root.**
+
+
+Install dependencies::
+
+    apt-get update
+
+    apt-get -y install unzip
+     
+    apt-get -y install python3
+    apt-get -y install python3-venv
+
+    apt-get -y install nginx
+
+
+Creating a virtual environment
+------------------------------
+
+**The following steps are run as root.**
+
+Create the virtual environment. This is the recommended way to install OMERO.web::
+
+    python3 -mvenv /opt/omero/web/venv3
+
+
+
+Install ZeroC IcePy 3.6::
+
+    /opt/omero/web/venv3/bin/pip install --upgrade https://github.com/ome/zeroc-ice-debian10/releases/download/0.1.0/zeroc_ice-3.6.5-cp37-cp37m-linux_x86_64.whl
+
+
+Install OMERO.web::
+
+    /opt/omero/web/venv3/bin/pip install "omero-web>=5.6.dev5"
+
+Installing OMERO.web apps
+-------------------------
+
+
+A number of apps are available to add functionality to OMERO.web, such as `OMERO.figure <https://www.openmicroscopy.org/omero/figure/>`_ and `OMERO.iviewer <https://www.openmicroscopy.org/omero/iviewer/>`_. See the main website for a `list of released apps <https://www.openmicroscopy.org/omero/apps/>`_. These apps are optional and can be installed, as the **root user**, via :program:`pip` to your OMERO.web virtual environment and configure as the **omero-web system user**, at any time.
```

Comment:
I don't understand how to configure as that user. Should this be "configured" and is it about "omero config set ..."?

Reply:
I actually found that quite clear. When you go through the installation you run `omero config ...` a few times, so at this step I'd naturally assume configure means `omero config...`.

Reply:
Maybe "yes" to both then, thank you.

Reply:
As discussed with @mtbc, I will add an example to avoid confusion

Reply:
resolved in omeroweb-install
Adding the example will mess the layout a bit so I will leave it for now